README.md

```diff
@@ -35,3 +35,15 @@ QuotaService quotaService = QuotaService.getReference("API_KEY");
 Product product = quotaService.getProductById("PRODUCT_ID");
 product.removeUser("USER_ID");
 ```
+### Increment a User's Quota
+```java
+QuotaService quotaService = QuotaService.getReference("API_KEY");
+Product product = quotaService.getProductById("PRODUCT_ID");
+User user = product.getUser("USER_ID");
+Quota quota = user.getQuota("QUOTA_ID");
+try {
+  quota.increment();
+} catch (QuotaReachedException e) {
```

Comment:
Doing this as an exception is pretty bad style.  We expect to go over quota and just need to know when it happens.
Probably fine for now though, add a TODO?

Reply:
In the future we will have a `isAtLimit()` check that developers woulf call before incrementing, so I don't think we would every really expect the exception to be thrown.

Reply:
I thought we were also going to make it return true or false so an alternative way they could code it is:
if (! incrementQuota()) {
   // handle that the user went over the quota
}

Reply:
It does return a boolean, but that boolean represents if the request failed due to say, our server being down instead of being over the quota.